pkg/cover/html.go

```diff
@@ -342,6 +342,10 @@ func fileContents(file *file, lines [][]byte, haveProgs bool) string {
 		}
 	}
 	buf.WriteString("</td><td>")
+	for i := range lines {
+		buf.WriteString(fmt.Sprintf("%d\n", i+1))
```

Comment:
I've tried to modify one of syzbot reports to what I think this PR will produce, and somehow I don't see line numbers at all... maybe I am doing something wrong. Do you see numbers with this change? Can you please attach some example coverage report?

Reply:
here's the one I build and tested just now

![image](https://user-images.githubusercontent.com/77719320/122385842-d5e16400-cf9f-11eb-8113-29c55e3cdc7e.png)

Reply:
and line number starts from 1

![image](https://user-images.githubusercontent.com/77719320/122386012-01644e80-cfa0-11eb-845b-546d15423033.png)

Reply:
OK, right, I see what I did wrong. I added only few line numbers and the column contents are centered vertically, so that they appeared somewhere in the middle of the file rather than at the top where I was looking for them.